libclc/opencl/include/clc/opencl/integer/abs.h

```diff
@@ -6,5 +6,12 @@
 //
 //===----------------------------------------------------------------------===//
 
+#ifndef __CLC_OPENCL_OPENCL_INTEGER_ABS_H__
```

Comment:
```suggestion
#ifndef __CLC_OPENCL_INTEGER_ABS_H__
```
one OPENCL should be sufficient, right?

Reply:
Oh, yes. And of course I'd copy/pasted the mistake to all files...

Reply:
Ah it's because I copied it incorrectly from `__CLC_OPENCL_OPENCL_BASE_H__` which does need it twice. Anyway, fixed. Thanks for spotting that!